.github/workflows/trigger-circle-ci-workflow.yml

```diff
@@ -56,7 +54,7 @@ jobs:
   trigger-pr-tests:
     runs-on: ubuntu-latest
     needs: get-branch
-    if: github.event_name == 'pull_request_target' && ((github.event.pull_request.draft == false || contains(github.event.pull_request.labels.*.name, 'ci:pr')) && !contains(github.event.pull_request.labels.*.name, 'ci:merged') && !contains(github.event.pull_request.labels.*.name, 'ci:daily'))
+    if: github.event_name == 'pull_request_target' && (((github.event.pull_request.draft == false && !contains(github.event.pull_request.labels.*.name, 'release')) || contains(github.event.pull_request.labels.*.name, 'ci:pr')) && !contains(github.event.pull_request.labels.*.name, 'ci:merged') && !contains(github.event.pull_request.labels.*.name, 'ci:daily'))
```

Comment:
trigger PR tests if:

1. is "pull_request_target" event
2. AND
	1. (not draft
	2. AND not "release")
	3. OR has "ci:pr" label
3. AND don't have "ci:merged"
3. AND don't have "ci:daily"